pkg/cloud/ovirt/clients/machineservice.go

```diff
@@ -360,10 +371,53 @@ func  (is *InstanceService) FindVirtualMachineIP(id string,excludeAddr map[strin
 
 				if hasAddress {
 					klog.Infof("ovirt vm id: %s , found usable IP %s", id, ipres)
-					return ipres,nil
+					return ipres, nil
 				}
 			}
 		}
 	}
 	return "", fmt.Errorf("coudlnt find usable IP address for vm id: %s", id)
```

Comment:
Typo :)

Reply:
Although it was added in a previous PR changed the VM name to be clearer :)

Reply:
I'm talking about "coudlnt"